api/refresh_continuous_aggregate.md

```diff
@@ -45,12 +45,12 @@ an `INTERVAL` type. For hypertables with integer-based timestamps, set the
 refresh window as an `INTEGER` type.
 
 <Highlight type="note">
-A `NULL` value of `window_start` is equivalent to the lowest changed element
-in the raw hypertable of the CAgg. A `NULL` value of `window_end` is
+A `NULL` value for `window_start` is equivalent to the lowest changed element
+in the raw hypertable of the CAgg. A `NULL` value for `window_end` is
 equivalent to the largest changed element in raw hypertable of the CAgg. Since
```

Comment:
As changed element tracking is performed after the initial CAgg refresh, CAgg refresh without `window_start` and
`window_end` covers the entire time range.